index.js

```diff
@@ -106,53 +109,51 @@ class instance extends instance_skel {
 	 * @since 1.0.0
 	 */
 	action(action) {
-		var cmd;
-		var opt = action.options;
+		var cmd
+		var opt = action.options
 
 		switch (action.action) {
 			case 'mode':
-				cmd = 'CONFIGURATION:\n'+'Solo enabled: '+ opt.mode +'\n\n';
-				break;
+				cmd = 'CONFIGURATION:\n' + 'Solo enabled: ' + opt.setting + '\n\n'
+				break
 			case 'audio':
-				cmd ='VIDEO OUTPUT ROUTING:\n'+(this.outputCount+1)+' '+opt.inp +'\n\n';
-				break;
+				cmd = 'VIDEO OUTPUT ROUTING:\n' + (this.outputCount + 1) + ' ' + opt.inp + '\n\n'
+				break
 			case 'solo':
-				cmd ='VIDEO OUTPUT ROUTING:\n'+this.outputCount+' '+opt.inp +'\n\n';
-				break;
+				cmd = 'VIDEO OUTPUT ROUTING:\n' + this.outputCount + ' ' + opt.inp + '\n\n'
+				break
 			case 'label':
-				let label = opt.label;
+				let label = opt.label
 				this.parseVariables(label, function (value) {
-					label = value;
-				});
-				cmd ='INPUT LABELS:\n'+ opt.inp +' '+ label +'\n\n';
-				break;
+					label = value
+				})
+				cmd = 'INPUT LABELS:\n' + opt.inp + ' ' + label + '\n\n'
+				break
 			case 'set_format':
-				cmd = "CONFIGURATION:\n"+"Output format: "+opt.setting+"\n\n";
-				break;
+				cmd = 'CONFIGURATION:\n' + 'Output format: ' + opt.setting + '\n\n'
+				break
 			case 'set_border':
-				cmd = "CONFIGURATION:\n"+"Display border: "+opt.setting+"\n\n";
-				break;
+				cmd = 'CONFIGURATION:\n' + 'Display border: ' + opt.setting + '\n\n'
+				break
 			case 'set_labels':
-				cmd = "CONFIGURATION:\n"+"Display labels: "+opt.setting+"\n\n";
-				break;
+				cmd = 'CONFIGURATION:\n' + 'Display labels: ' + opt.setting + '\n\n'
+				break
 			case 'set_meters':
-				cmd = "CONFIGURATION:\n"+"Display audio meters: "+opt.setting+"\n\n";
-				break;
+				cmd = 'CONFIGURATION:\n' + 'Display audio meters: ' + opt.setting + '\n\n'
+				break
 			case 'set_tally':
-				cmd = "CONFIGURATION:\n"+"Display SDI tally: "+opt.setting+"\n\n";
-				break;
+				cmd = 'CONFIGURATION:\n' + 'Display SDI tally: ' + opt.setting + '\n\n'
+				break
 			case 'set_widescreen_sd':
-				cmd = "CONFIGURATION:\n"+"Widescreen SD enable: "+opt.setting+"\n\n";
-				break;
+				cmd = 'CONFIGURATION:\n' + 'Widescreen SD enabled: ' + opt.setting + '\n\n'
```

Comment:
This fixes it for me.

Obviously I can't tell whether they've changed it over the years (like the output format), or it wasn't tested sufficiently initially (like the solo mode). We could add in some other stuff to send and receive both variants, but maybe that's excessive?

Reply:
It's not been fixed/still says enable all over the place in the Multiview16 too...

Reply:
The manual says enable for version 2.3 of the protocol, I'm on 2.8...

Reply:
The code now handles both variants just in case...